pkgs/os-specific/linux/zfs/default.nix

```
@@ -202,7 +202,7 @@ let
         changelog = "https://github.com/openzfs/zfs/releases/tag/zfs-${version}";
         license = licenses.cddl;
         platforms = platforms.linux;
-        maintainers = with maintainers; [ hmenke jcumming jonringer wizeman fpletz globin mic92 ];
+        maintainers = with maintainers; [ hmenke jcumming jonringer wizeman fpletz globin ];
```

I don't think @fpletz` and @globin maintaining realistically.

```suggestion
        maintainers = with maintainers; [ hmenke jcumming jonringer wizeman ];
```

---

I am not sure about this one but the PR itself gets a ✅ from me.